### ISSUE TITLE
Storefront: Create blog post and blog archive layouts

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#1423]
```[tasklist]
### Tasks
- [x] Design blog post layout
- [x] Design blog archive layout
- [x] Make Banner component before starting development on this
- [ ] https://github.com/digdir/designsystemet/issues/1539
- [x] Create dummy pages for both and hide them when the code is merged into main.
```

**Comment by Thuneer** [comment]
The blog tasks are done and has been implemented on the Storefront.
